<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>send_img_dwt_fountain</class>
 <widget class="QDialog" name="send_img_dwt_fountain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_packet_interval">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_send_image_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>219</y>
     <width>330</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>send status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_packet_interval">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Packet Interval</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_send_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
     border-width: 1px;
     border-style: solid;
     border-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>119</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_header_wait_time">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Header Wait Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_header_wait_time">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_w1_size">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_w1_para">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>w1 para</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_w1_pro">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ready">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
